my/core/query_range.py

```diff
@@ -73,13 +73,28 @@ def parse_datetime_float(date_str: str) -> float:
         return ds_float
     try:
         # isoformat - default format when you call str() on datetime
+        # this also parses dates like '2020-01-01'
         return datetime.fromisoformat(ds).timestamp()
     except ValueError:
         pass
     try:
         return isoparse(ds).timestamp()
     except (AssertionError, ValueError):
-        raise QueryException(f"Was not able to parse {ds} into a datetime")
+        pass
+
+    try:
+        import dateparser # type: ignore[import]
```

Comment:
when I used dateparser in the past, I'd usually configure it via settings to prefer YMD/DMY order -- by default it's locale dependent which may be an issue, e.g.

```
dateparser.parse('04-05-2021')
Out[3]: datetime.datetime(2021, 4, 5, 0, 0)
```
which is odd even though I have `en_GB` locale (so I'd expect the default to be DMY). But dunno what's a good default really

Reply:
Probably a good idea to make it `YMD` by default since thats what we show in examples, and so its possibly easier to debug if someone is getting incorrect dates